sdk/samples/keyvault/keyvault/samples/src/keys_client_example.c

```diff
@@ -43,7 +43,6 @@
 #include <az_context.h>
 #include <az_credentials.h>
 #include <az_http.h>
-#include <az_http_internal.h>
 #include <az_json.h>
 #include <az_keyvault.h>
 
```

Comment:
RETURN_IF_FAILED(.......) is a better name for the TEST_FAIL_ON_ERROR macro

Reply:
Here is what I propose: https://github.com/Azure/azure-sdk-for-c/pull/758